src/screens/ProfileScreen/styles.ts

```diff
@@ -61,11 +60,11 @@ export const profileScreenStyles = StyleSheet.create({
   },
   stickyButtonContainer: {
     position: 'absolute',
-    bottom: 16, // Adjust the value based on your preference
-    left: '20%', // Center the button horizontally
-    transform: [{ translateX: -50 }], // Center the button horizontally
-    width: '80%', // Adjust the width based on your preference
-    zIndex: 2, // Ensure the button is above other elements
+    bottom: 16,
```

Comment:
dont hard code 16 use scale function, should be responsive

Reply:
for this also, we need to add a new issue  as we have to update it everywhere with testing.